Cargo.toml

```diff
@@ -25,10 +25,10 @@ default = ["ledger_device_sdk/nano_nbgl"]
 debug = ["ledger_device_sdk/debug"]
 
 [package.metadata.ledger]
-curve = ["secp256k1"]
+curve = ["ed25519"]
 flags = "0"
-path = ["44'/1'"]
-name = "Rust Boilerplate"
+path = ["44'/457'"]
+name = "aeternity"
```

Comment:
It was "Aeternity" before, but the lowercase look more consistent with our branding. I also tried "æ", but Ledger Nano S doesn't have such a glyph.